src/ClientBundle/Form/Handler/AbstractClientFormHandler.php

```
@@ -47,7 +47,7 @@ public function __construct(RouterInterface $router)
 
     public function getForm(FormFactoryInterface $factory, Options $options): FormInterface
     {
-        return $factory->create(ClientType::class, $options->get('client', new Client()));
+        return $factory->create(ClientType::class, $options->get('client') ?? new Client());
     }
 
     public function onSuccess(FormRequest $form, $contact): ?Response
```

PHPMD (2.15.0)

Avoid unused parameters such as '$form'. (undefined)

(UnusedFormalParameter)